test/helper.py

```diff
@@ -124,8 +124,8 @@ def has_program(cmd, args=['--version']):
     """
     full_cmd = [cmd] + args
     for i, elem in enumerate(full_cmd):
-        if isinstance(elem, six.text_type):
-            full_cmd[i] = elem.encode(_arg_encoding())
+        if isinstance(elem, bytes):
+            full_cmd[i] = elem.decode(_arg_encoding())
```

Comment:
if this is the approach we wanna take then yes, but I wondered if you had any better ideas.